### PR TITLE
chore: change release notes template, auto label PRs, ...

### PR DESCRIPTION
**Pull request by hilmarf**

### CODE REVIEW

**Review by morri-son — APPROVED** [review]
gefällt mir gut :-)

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+labels: kind/skip-release-notes
```
Comment (frewilhelm) [inline .github/pull_request_template.md:2]: Are all the PRs now labeled with `skip-release-notes`?

Comment (hilmarf) [reply .github/pull_request_template.md:2]: will test this soon and let you know

**Comment by hilmarf** [comment]
related https://github.com/open-component-model/ocm-project/issues/24

**Pull request merged**